one-pager-maker/src/App.tsx

```diff
@@ -42,16 +43,16 @@ function App() {
                                     <Link to="/edit">編集</Link>
                                 </li>
                                 <li>
-                                    <Link to="/login">ログイン</Link>
+                                    <Button onClick={() => { signOut(auth) }}>ログアウト</Button>
                                 </li>
                             </ul>
                         </nav>
                         <Router />
-                    </BrowserRouter>
-                </>
-            ) : (
-                <Login />
-            )}
+                    </>
+                ) : (
+                    <SignInRouter />
```

Comment:
Typically one react website has only one Router as long as I remember.
So consider refactoring next time and use AuthGuard for each Route. 
I'll show you an example someday.